pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java

```diff
@@ -339,7 +328,8 @@ public ClientBuilder enableTransaction(boolean enableTransaction) {
 
     @Override
     public ClientBuilder dnsLookupBind(String address, int port) {
-        checkArgument(port >= 0 && port <= 65535, "DnsLookBindPort need to be within the range of 0 and 65535");
+        checkArgument(port >= 0 && port <= 65535,
```

Comment:
is that a unnecessary format?

Reply:
it's too long,  I split into two line by the way

Reply:
do I need to recover ?

Reply:
recovered!